src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (...args) => {
+    if (args.length > 0) {
+      sum += args.reduce((acc, val) => acc + val, 0);
+
+      return adder;
+    } else {
+      const result = sum;
+
+      sum = 0;
+
```

Comment:
This line is unnecessary and can be removed to improve code clarity. The logic will still work correctly without it.